packages/client/src/pages/settings/preferences-registry.vue

```diff
@@ -0,0 +1,380 @@
+		if (defaultStore.def[key].where !== 'account') acc[key] = defaultStore.state[key];
+		return acc;
+	}, {} as any);
+}
+
+function isObject(value: any) {
+	return value && typeof value === 'object' && !Array.isArray(value);
+}
+
+function validate(registry: any): void {
+	if (!registries) return;
+
+	// Check if unnecessary properties exist
+	if (Object.keys(registry).some(key => !registryProps.includes(key))) throw Error('Unnecessary properties exist');
+
+	if (!registry.name) throw Error('Name is falsy');
+	if (!registry.version) throw Error('Version is falsy');
+	
+	// Check if createdAt and updatedAt is Date
+	// https://zenn.dev/lollipop_onl/articles/eoz-judge-js-invalid-date
+	if (!registry.createdAt || Number.isNaN(new Date(registry.createdAt).getTime())) throw Error('createdAt is falsy or not Date');
+	if (registry.updatedAt) {
+		if (Number.isNaN(new Date(registry.updatedAt).getTime())) {
+			throw Error('updatedAt is not Date');
+		}
+	} else if (registry.updatedAt !== null) {
+		throw Error('updatedAt is not null');
+	}
+
+	if (!registry.defaultStore || !isObject(registry.defaultStore)) throw Error('defaultStore is falsy or not an object');
+	if (!registry.coldDeviceStorage || !isObject(registry.coldDeviceStorage)) throw Error('coldDeviceStorage is falsy or not an object');
+}
+
+async function saveNew() {
+	if (!registries) return;
+
+	const { canceled, result: name } = await os.inputText({
+		title: ts._preferencesRegistry.inputName,
+		text: ts._preferencesRegistry.saveNewDescription,
+	});
+
+	if (canceled) return;
+	if (Object.entries(registries).some(e => e[1].name === name)) {
+		return os.alert({
+			title: ts._preferencesRegistry.cannotSave,
+			text: t('_preferencesRegistry.nameAlreadyExists', { name }),
+		});
+	}
+
+	const id = uuid();
+	const registry: Registry = {
+		name,
+		createdAt: (new Date()).toISOString(),
+		updatedAt: null,
+		version,
+		defaultStore: getDefaultStoreValues(),
+		coldDeviceStorage: ColdDeviceStorage.getAll(),
+		fontSize: localStorage.getItem('fontSize'),
+		useSystemFont: localStorage.getItem('useSystemFont') as 't' | null,
+		wallpaper: localStorage.getItem('wallpaper'),
+	};
+	await os.apiWithDialog('i/registry/set', { scope, key: id, value: registry });
+}
+
+function loadFile() {
+	const input = document.createElement('input');
+	input.type = 'file';
+	input.multiple = false;
+	input.onchange = async () => {
+		if (!registries) return;
+		if (!input.files || input.files.length === 0) return;
+
+		const file = input.files[0];
+
+		if (file.type !== 'application/json') {
+			return os.alert({
+				type: 'error',
+				title: ts._preferencesRegistry.cannotLoad,
+				text: ts._preferencesRegistry.invalidFile,
+			});
+		}
+
+		let registry: Registry;
+		try {
+			registry = JSON.parse(await file.text()) as unknown as Registry;
+			validate(registry);
+		} catch (e) {
+			return os.alert({
+				type: 'error',
+				title: ts._preferencesRegistry.cannotLoad,
+				text: e?.message,
+			});
+		}
```

Comment:
Using `e` as an identifier is not allowed, this adds lint errors (for errors, prefer `err` instead)

Reply:
```suggestion
		} catch (err) {
			return os.alert({
				type: 'error',
				title: ts._preferencesRegistry.cannotLoad,
				text: err?.message,
			});
		}
```